subresource-loading.md

```diff
@@ -237,20 +244,38 @@ When returning to the event loop, if there are any chunk IDs on the list to fetc
 - When the fetch returns from the server, map in all fetched chunk IDs and add them to the persistent cache.
 - Then, the original fetch can continue at the top of the algorithm. It should either be served by a mapped-in chunk or 404.
 
+#### Pros
+* Mixing in decelration of the resources in the bundle and scopes covered by it in ways that enable use of both techniques, without requiring hard-to-implement [Cache Digests](https://tools.ietf.org/html/draft-ietf-httpbis-cache-digest-05).
+* Decouples the resource number from the number of chunks in ways that enable higher scalability in cases of closely knit, very small modules.
+* Update flow doesn't require content-addressable URLs, which avoids problems with unnecessary cache invalidation of references, as well as "purge" like semantics when resource version changes.
+  - Unclear if that part would work properly in the "online URL enforcement" revalidation flow.
+
+#### Cons
+* Requires declaring full dependency map on each entry point, that includes all the chunks required by all the entry points. If developers will get that wrong, it can result in spurious downloads.
+* A "just download the entire bundle" fallback is not feasible, since the full bundle can be extremely large. Related, caches that ignores Vary will have have A Bad Time™. 
```

Comment:
To be more specific than "extremely large", you could say here, "since the full bundle contains all of the code-split parts, but initial load should only load the necessary parts".

(Are there caches that ignore Vary? Yeah, it does seem like they will have a bad time...)

Reply:
That phrasing WFM.

There are indeed caches that ignore Vary (in varying degrees :P) , and they'd need to figure out alternative ways to handle this.

Reply:
Oh no! That seems quite bad. Should we put the chunk list as part of the URL instead (in a query parameter), to accommodate them?